app/views/generic_files/_versioning.html.erb

```diff
@@ -21,9 +21,11 @@
     <% @version_list.each do |version| %>
       <div class="form-group">
         <div class="col-xs-12">
-          <label class="radio">
-            <%= radio_button_tag(:revision, version.label, version.current?) %>
-            Restore from <%= version.created %> [by <%= version.committer %>]
+          <% disable_if_current_version = version.current? ? 'disabled' : '' %>
+          <label class="radio <%= disable_if_current_version %>">
```

Comment:
Not sure I like the layer of indirection here.  Why not just use version.current? ? 'disabled' : '' in the label.  disable_if_current_version is only being used here correct?